.env

```diff
@@ -0,0 +1,3 @@
+USER_EMAIL = smtp.gmail.com
+EMAIL_PASSWORD = agroeasy1#$##
+EMAIL_HOST = agroeasy2018@gmail.com
```

Comment:
Is this meant to be pushed

Reply:
no it is not

Reply:
@joshuaobinna remove this files from your commits